src/lib/util.py

```diff
@@ -2,9 +2,15 @@ def list_intersect(list_a, list_b):
   ''' return elements common between l1 and l2 '''
   return [ x for x in list_a if x in list_b]
 
-def list_subtract(list_a, list_b):
-  ''' return elements that are in list a but not b '''
-  return [x for x in list_a if x not in list_b]
+def list_subtract(list_a, *args):
+  '''
+  return elements that are in list a but
+  not in any list supplied by *args
+  '''
+  minuend = list_a
+  for subtrahend in args:
+    minuend = [x for x in minuend if x not in subtrahend]
+  return minuend
```

Comment:
weird var names but apparently for

a - b

a is called minuend
b is called subtrahend

Reply:
Thing I learned!
![parts-of-subtraction-problem_7abbbb2796](https://github.com/artsy/opstools/assets/28120/1c520a43-82a8-4363-8c5b-ca9b49487ae6)